crates/graphql_network_protocol/src/graphql_network_protocol.rs

```diff
@@ -234,6 +242,25 @@ impl NetworkProtocol for GraphQLNetworkProtocol {
             {indent}}}"
         ))
     }
+
+    fn get_id_field_name(
+        db: &IsographDatabase<GraphQLNetworkProtocol>,
+        server_object_entity_name: &ServerObjectEntityName,
```

Comment:
Btw you should generally never write a function that takes a reference to a `Copy` struct (and any interned strings are Copy). (One exception is if you need to because you're implementing a trait and the trait requires &T, or if the struct is copy but really really big. Anyway, these aren't big), so let's accept `server_object_entity_name: ServerObjectEntityName`.